core/lib/libmbedtls/src/tee_lmd_provider.c

```diff
@@ -198,6 +246,7 @@ TEE_Result crypto_hash_update(void *ctx, uint32_t algo,
 	default:
 		return TEE_ERROR_NOT_SUPPORTED;
 	}
+
```

Comment:
Unrelated change

Reply:
Yes, sorry about that. I will fix it too.